cmd/varint/doc.go

```diff
@@ -0,0 +1,28 @@
+/*
+
+Command varint encodes a decimal number to or from varint.
+
+Usage:
+
+      varint [-u] [-s]
+
+It takes flags -s and -u to choose signed or unsigned encoding.
+The default is unsigned (and unsigned is the only encoding used in Chain Protocol).
+
+It reads from stdin when decoding, and takes a parameter when encoding.
+
+Examples:
+
+Obtain the decimal value of the hex-encoded varint 0101:
+
+      printf 0101 | hex -d | varint -u
+
+Obtain the unsigned varint value of the decimal number 1234:
+
+      varint -u 1234 | hex -e
```

Comment:
nit: I would leave off flags for default behavior in general (so both flags here and -u on line 18)